### ISSUE TITLE
Opening an image from Submission Screen does not seem to honoro data saving setting

### ISSUE THREAD

**Issue by ianrobrien** [ccrama/Slide-iOS#765]
When I click an image from the submission (main screen) it does not seem to respect the data saving option. The image has no "HD" option on the bottom bar. When I click into the submission and open the image, I see the option for "HD". This leads me to believe that the option is not respected from the submissions screen. I am not connected to WiFi.

- My Settings
  - Submission Layout
    - Submission view mode: Card
    - Banner images: Thumbnail
  - Data Saving
    - Data saving enabled
    - Data saving on WiFi

Steps to reproduce

1. Click an image from the thumbnail on the submission screen
2. Notice that it does not have an "HD" button on the bottom
3. Navigate into the post itself by clicking the posts title
4. From the post itself, click on the image
5. Notice that the image has an "HD" button on the bottom
6. If you click the "HD" button a higher quality image is loaded

Expected behavior (with data savings enabled)

1. Click on the image from the thumbnail on the submission screen
2. The "HD" button should be visible and a lower quality image should be loaded

**Comment by ccrama** [comment]
When you are noticing this behavior, are you opening it from a thumbnail on the post screen and the large banner image from comments?

**Comment by ianrobrien** [comment]
Yes, that's correct. Opening from the thumbnail on the post screen seems to always open the HD version of the image. If I go into the post itself (by clicking the title), then opening the image there seems to show first the low-quality version, with the option for HD on the bottom.
